src/components/organisms/PaneManager/PaneManager.tsx

```diff
@@ -23,14 +34,49 @@ const PaneManager: React.FC = () => {
     return 'max-content 1fr';
   }, [activeProject, isStartProjectPaneVisible]);
 
+  const handleBottomPaneResize = useCallback(
+    (elements: any) => {
+      const bottomPaneElement = elements.find((el: any) => el.props.id === 'bottomPane');
```

Comment:
what if `bottomPaneElement` will be `undefined` ? maybe the case should be handled